resources/kiali/templates/tests/test-kiali-connection.yaml

```diff
@@ -0,0 +1,28 @@
+
+{{- if .Capabilities.APIVersions.Has "testing.kyma-project.io/v1alpha1" }}
+apiVersion: "testing.kyma-project.io/v1alpha1"
+kind: TestDefinition
+metadata:
+  name: kiali
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: kiali-test
+    chart: {{ template "kiali.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  disableConcurrency: false
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "false"
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: "{{ template "kiali.fullname" . }}-test"
+        image: {{ .Values.global.containerRegistry.path }}/{{ .Values.global.alpine_net.dir }}/{{ .Values.global.alpine_net.name }}:{{ .Values.global.alpine_net.version }}
+        imagePullPolicy: "{{ .Values.global.imagePullPolicy }}"
+        command: ['curl']
```

Comment:
If the test only curl kiali to see if it alive, better remove the test altogether and change this to a readiness check

Reply:
I mainly want to assure that the full exposure is working as here most of the problems occur. 
I mainly adopted the test from the kiali chart itself and I think we should extend it in future to check that basic functionality is availble.